.travis.yml

```diff
@@ -16,6 +16,10 @@ addons:
     - mysql-client-core-5.6
     - mysql-client-5.6
 
+branches:
+  only:
+    - master
```

Comment:
sry for this offtopic change, but building the branch AND the PR with Travis slows down everything, so master should be the only branch to be built